spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/ImportExportControllerHelperService.java

```diff
@@ -570,7 +572,7 @@ public JSONObject librarySelectionTypesString(HttpSession session, JSONObject js
 
   public JSONObject changePlatformName(HttpSession session, JSONObject json) {
     try {
-      if (json.has("platform") && !json.get("platform").equals("")) {
+      if (json.has("platform") && !isStringEmptyOrNull((String) json.get("platform"))) {
```

Comment:
I'm beginning to think maybe `isStringEmptyOrNull` could take care of the cast for you.  What do you reckon?